### DISCUSSION TITLE
Custom `Debug` implementation for `User` but not `Credentials` in SQLite example?

### DISCUSSION THREAD

**Discussion by homeworkprod** [maxcountryman/axum-login#130] [Q&A]
`examples/sqlite/src/users.rs` has a custom `Debug` implementation for `User` "to avoid accidentally logging the password hash."

`Credentials` has a field `password`, too, but derives `Debug`.

Shouldn't `Credentials` have a custom `Debug` implementation that redacts the password as well by that logic?

**Comment by maxcountryman** [comment]
We could also use something like [`redact`](https://crates.io/crates/redact).

Updating the example with a custom `Debug` impl is a great starting place if that's easier.
